src/main/java/it/gov/pagopa/rtd/ms/rtdmsfileregister/domain/DefaultNamingPolicy.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.rtd.ms.rtdmsfileregister.domain;
+
+import it.gov.pagopa.rtd.ms.rtdmsfileregister.model.FileType;
+import org.springframework.data.util.Pair;
+
+import java.util.Optional;
+
+public class DefaultNamingPolicy implements NamingConventionPolicy {
+  @Override
+  public String extractParentFileName(String filename, FileType fileType) {
+    if (fileType == FileType.TRANSACTIONS_SOURCE
+            || fileType == FileType.AGGREGATES_SOURCE
+            || fileType == FileType.ADE_ACK
+    ) {
+      return filename;
+    }
+    if (fileType == FileType.AGGREGATES_CHUNK) {
+      final var parts = filename.split("\\.");
+      final var batchChunk = extractBatchServiceChunk(parts).orElse("");
+      return "ADE." + parts[1] + ".TRNLOG." + parts[2] + "." + parts[3] + "." + parts[4]
+              + batchChunk
+              + ".csv.pgp";
+    }
+    if (fileType == FileType.TRANSACTIONS_CHUNK) {
+      return filename.replaceAll("\\.(\\d)+\\.decrypted", "");
+    }
+    if (fileType == FileType.AGGREGATES_DESTINATION) {
+      String[] parts = filename.replace(".gz", "").split("\\.");
+      final var batchChunk = extractBatchServiceChunk(parts).orElse("");
+      //return "AGGADE." + parts[1] + "." + parts[2] + "." + parts[3] + "." + parts[4] + ".csv.pgp";
+      return "ADE." + parts[1] + ".TRNLOG." + parts[2] + "." + parts[3] + "." + parts[4]
+              + batchChunk
+              + ".csv.pgp";
+    }
+    if (fileType == FileType.SENDER_ADE_ACK) {
```

Comment:
the fix on this field will be done in a different task?

Reply:
Which fix? `AGGREGATES_DESTINATION` file types?

Reply:
`SENDER_ADE_ACK`, atm do not retrieve correctly the parent name. Maybe the activity can be tracked on a different task.